static/src/javascripts/boot.js

```diff
@@ -12,20 +12,19 @@ Now we can safely require one of those modules.
 Unfortunately we can't do all of this using the curl API, so we use a
 combination of ajax/eval/curl instead.
 
-Bundles we need to run: commercial + (enhanced-vendor + enhanced)*
-* Only if we detect we should run enhance
+Bundles we need to run: commercial + enhanced
+
+Only if we detect we should run enhance.
  */
 
 define([
     'Promise',
```

Comment:
do these methods all really need to be wrapped in promises? there's not much going on now, feel a bit like overkill?